.github/workflows/testing.yml

```diff
@@ -103,10 +104,18 @@ jobs:
           go version
           go test ${{ matrix.testflags }} -cpu 1,4 -timeout 7m google.golang.org/grpc/...
           cd "${GITHUB_WORKSPACE}"
+          set -x
           for MOD_FILE in $(find . -name 'go.mod' | grep -Ev '^\./go\.mod'); do
             pushd "$(dirname ${MOD_FILE})"
-            go test ${{ matrix.testflags }} -cpu 1,4 -timeout 2m ./...
+            # Skip OpenTelemetry module if 3 releases ago, Go OpenTelemetry only supports
+            # previous two releases.
+            if [[ "${{ matrix.skipotel }}" == "true" && "${PWD}" =~ /stats/opentelemetry$ ]]; then
```

Comment:
Maybe we should name this something nicer so the value (which is an arbitrary string not a boolean) doesn't say "true".

I.e.

```
- type: tests
  goversion: '1.20'
  mode: skipotel
```

Reply:
Done.